libs/ui-lib/lib/cim/types/k8s/agent-cluster-install.ts

```diff
@@ -19,6 +19,9 @@ export type AgentClusterInstallK8sResource = K8sResourceCommon & {
     clusterDeploymentRef?: {
       name: string;
     };
+    manifestsConfigMapRefs?: {
```

Comment:
I realized that this is not supposed to reference `MachineConfig` but `ConfigMap` that can contain various `yamls` (including the MachineConfigs)